### DISCUSSION TITLE
SearchInput outside click overlay wrong z-index

### DISCUSSION THREAD

**Discussion by oleghalin** [laravel/nova-issues#4038] [Q&A]
- Laravel Version: 9.8.1
- Nova Version: 4.2.2
- PHP Version: 8.1.1
- Browser type and version: Chrome 

### Description:
New SearchInput component includes overlay which used for click-outside logic. This overlay has wrong z-index (z-45). It conflicts with other z-index values and doesn't allow to select items from SearchInput select when SearchInput located on Resource index page. 

![image](https://user-images.githubusercontent.com/9086983/163986985-b52a9d38-8bf1-4333-8a53-a864d20bb27d.png)

It should be changed to z-35 and issue will be solved.

**Comment by crynobone** [comment]
![Screenshot_20220419-185600_Chrome](https://user-images.githubusercontent.com/172966/163989106-09c9e66f-e10e-41cd-bd39-d883ddcd47c6.jpg)

Where exactly is the issue. You can't just say you suggestion solve an issue without explicitly explaining the issue.

**Comment by oleghalin** [comment]
@crynobone `vendor/laravel/nova/resources/js/components/Inputs/SearchInput.vue:110`

**Comment by oleghalin** [comment]
Your z-index grid doesn't assume that SearchInput will be used inside of filter dropdown 
![image](https://user-images.githubusercontent.com/9086983/163990116-7aea6ed1-0c35-4490-a955-d653ef016129.png)

**Comment by crynobone** [comment]
Look right to me, Dropdown has z-index 40 so SearchInput backdrop z-index should be higher.

**Comment by oleghalin** [comment]
Dropdown z-index is 40, while backdrop for SearchInput z-index is 45. So it covers DropDown
